src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -131,8 +131,10 @@
   --#{$menu-toggle}--m-plain--BorderColor: transparent;
   --#{$menu-toggle}--m-plain--hover--BackgroundColor: var(--pf-t--global--background--color--action--plain--hover);
   --#{$menu-toggle}--m-plain--expanded--BackgroundColor: var(--pf-t--global--background--color--action--plain--clicked);
-  --#{$menu-toggle}--m-plain--disabled--Color: var(--pf-t--global--icon--color--disabled); // picking icon color rather than text...?
-  --#{$menu-toggle}--m-plain--disabled--BackgroundColor: transparent; // picking icon color rather than text...?
+  --#{$menu-toggle}--m-plain--disabled--Color: var(--pf-t--global--text--color--disabled);
+  --#{$menu-toggle}--m-plain--disabled__icon--Color: var(--pf-t--global--icon--color--disabled);
+  --#{$menu-toggle}--m-plain--disabled__toggle-icon--Color: var(--pf-t--global--icon--color--disabled);
+  --#{$menu-toggle}--m-plain--disabled--BackgroundColor: transparent;
```

Comment:
Another request from design, noticed a bug where plain toggle icons were using `--on-disabled` and they should just be using `--disabled`